spec/functional/x_cache_headers_spec.rb

```diff
@@ -15,24 +16,48 @@
     expect(DOMAIN + '/correct-true-cache-key').to have_cp_code('cp-code')
   end
 
+  it 'should succeed when the cp code set in x-true-cache-key in the rest response' do
```

Comment:
I think this would read better as `should succeed when the correct cp code is in the response`